### PR TITLE
Unit tests for team link and positions endpoints

### PR DESCRIPTION
**Pull request by AndriiS1**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Team/TeamMemberLink/GetAllTeamMembersLinksTest.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        }
+
+        [Fact]
+        public async Task ShouldReturnSuccessfully_WhenCountMatch()
+        {
+            //Arrange
+            _mockRepository.Setup(x => x.TeamLinkRepository.GetAllAsync(
+                null,
+                It.IsAny<Func<IQueryable<TeamMemberLink>, IIncludableQueryable<TeamMemberLink, object>>>()))
+                .ReturnsAsync(GetTeamMemberLinksList());
```
Comment (KateYatsiuk) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Team/TeamMemberLink/GetAllTeamMembersLinksTest.cs:60]: The same problem with duplications of setup here

**Review by KateYatsiuk — CHANGES_REQUESTED** [review]

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Team/Position/GetAllPositionTest.cs [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        {
+            new Positions
+            {
+                Id = 1
+            },
+
+            new Positions
+            {
+                Id = 2
+            }
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.XUnitTest/MediatRTests/Team/Position/GetAllPositionTest.cs:115]: code styles, add tabs

**Review by Tysyatsky — COMMENTED** [review]
Left some comments, correct them pls.

File: Streetcode/Streetcode.XUnitTest/MediatRTests/Team/Position/CreatePositionTest.cs [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+            _mockMapper.Setup(x => x.Map<Positions>(It.IsAny<PositionDTO>()))
+                .Returns(testPositions);
+            _mockMapper.Setup(x => x.Map<PositionDTO>(It.IsAny<Positions>()))
+                .Returns(GetPositionsDTO());
+
+            _mockRepository.Setup(x => x.PositionRepository.CreateAsync(It.Is<Positions>(y => y.Id == testPositions.Id)))
+                .ReturnsAsync(testPositions);
+            _mockRepository.Setup(x => x.SaveChanges())
+                .Returns(1);
```
[the remark this answers is not in the document]
Comment (AndriiS1) [reply Streetcode/Streetcode.XUnitTest/MediatRTests/Team/Position/CreatePositionTest.cs]: Thanks, solved

**Review by KateYatsiuk — APPROVED** [review]

**Review by DanyilTerentiev — APPROVED** [review]
Great work! I like how precise you covered all the cases

**Comment by AndriiS1** [comment]
> Great work! I like how precise you covered all the cases 

Glad to hear that

**Review by Tysyatsky — APPROVED** [review]
looks great.

**Pull request merged**